controller/src/test/java/io/pravega/controller/server/ControllerServiceWithStreamTest.java

```diff
@@ -65,6 +65,10 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Nit: we should remove * in import ref: https://github.com/pravega/pravega/blob/master/checkstyle/checkstyle.xml#L78